test/integration/cms/api/templates_test.rb

```diff
@@ -95,6 +95,27 @@ def setup
         assert_equal 2, response.parsed_body['collection'].size
       end
 
+      test 'index filters by type' do
+        5.times { FactoryBot.create(:cms_page, provider: @provider) }
+        2.times { FactoryBot.create(:cms_partial, provider: @provider) }
```

Comment:
Last one, I promise:
```suggestion
        FactoryBot.create(:cms_page, 5, provider: @provider)
        FactoryBot.create(:cms_partial, 2, provider: @provider)
```